CHANGES.md

```diff
@@ -1,5 +1,23 @@
 # Changes
 
+## edge-23.6.1
+
+This edge release changes the behavior of the CNI plugin to run exclusively in
+"chained mode". Instead of creating its own configuration file, the CNI plugin
+will now wait until a `conf` file exists before appending its configuration.
+Additionally, this change includes a bug fix for topology aware service
+routing.
+
+* Changed CNI plugin installer to always run in 'chained' mode; the plugin will
+  now wait until another CNI plugin is installed before appending its
+  configuration
+* Added a timeout value to the HttpRoute CRD. The field is currently unused but
+  will be used eventually to allow for configuration of per-route timeouts
+* Fixed bug where topology routing would not disable while service was under
```

Comment:
```suggestion
* Fixed a bug where topology-aware routing would not be disabled while a Service was under
```

Reply:
These all make sense to me. I already included the notes in the stable though (https://github.com/linkerd/linkerd2/blob/stable-2.13.4/CHANGES.md) 🙃 might just leave it as is so it's consistent.